acceptance/tests/service_enable_start.rb

```diff
@@ -0,0 +1,12 @@
+test_name 'Validate that pxp-agent service can be enabled and started'
```

Comment:
Not critical; but for the other tests I've been putting the Test Rail Case ID (C12345) at the start of the test_name